js/chart.js

```diff
@@ -0,0 +1,220 @@
+// Filter out an array with the users in past 60 minutes
+function filter(data) {
+  //this gives you an array of tuples where the first element is the hour and the last element is the # of users
+  var result = data.rows.map(
+    function(elem) {
+      return [elem[0], elem[1], elem[2]]
```

Comment:
given that this project has only two js files, it would be ideal if they both either use semicolons or don't